exampleCourse/serverFilesCourse/templates/collapsible_section.mustache.html

```diff
@@ -0,0 +1,18 @@
+<div class="card mb-1 mt-1">
+  <div class="card-header" style="cursor: pointer">
+    <h4
+      class="card-title d-flex justify-content-between"
+      data-toggle="collapse"
+      data-target="#{{section_name}}"
+    >
+      <div>{{section_header}}</div>
+      <div class="fa fa-angle-down"></div>
+    </h4>
+  </div>
+
+  <div class="collapse{{#show}} show{{/show}}" id="{{section_name}}">
```

Comment:
I get that you're just trying to illustrate using templates, but this feels like in the "real world" this would make much more sense as an element that can automatically assign an ID here. As written, this question has a very strange UX: as soon as someone submits an answer to this question, there will be two panels with identical IDs, which is a) not allowed and b) means that every instance of this panel will expand/collapse whenever any instance is clicked.

Is there a more realistic use case for templates that we could use instead?

Reply:
So this actually does come from a real world example (donated by @mfbutner), but I think you're right that this would make a bit more sense as an element. The `pl-template` element lives in an unfortunate middle-ground where it's very useful for people that just want to have boilerplate text copied, but for those that don't want to write their own elements, it's not _quite_ flexible enough that it's easy to get yourself into trouble with stuff like this.

Thinking about it some, there's enough HTML like this that relies on having a unique id that it's worth having the `pl-template` always contain a `uuid` generated for use just by the contents of this template. I'll update this example and the documentation accordingly.

Reply:
Sorry for being somewhat absent from this conversation. Regarding the example I submitted. 
1. I am very new to writing stuff for the front end so I probably made quite a few mistakes or did things a weird/wrong way.
2. I didn't notice the multiple id issue because I had always been wrapping my templates at the highest level in a `<pl-question-panel>`. So since it rendered only in the question panel there weren't multiple ids. I shouldn't be expecting everyone else to be using it  this same way so I'll probably have to update the example to make it more universal
3. I didn't consider writing my own custom element because up until reading your message I only considered writing an element if I needed some custom grading logic along with custom displaying. I forgot you could write elements that were decorative only even though there are already a ton of them. I noticed what eliot had made and it seemed like exactly what I needed: a way to easily copy and paste around the same block of text with some minor variations across multiple questions. It has been really helpful and helped to cut down on a lot of code bloat.

Reply:
So in the latest pushes, I added the `uuid` generation for each separate `<pl-template>` and updated the examples, so the duplicate IDs should no longer be an issue @mfbutner.

To your third item: This is actually what I like most about the `pl-template` as a concept; instructors can essentially write their own decorative elements with this without the need to define a custom element each time, which cuts down heavily on code duplication.